spec/features/visitors/statistics.rb

```diff
@@ -0,0 +1,10 @@
+feature 'Statistics page' do
+
+  scenario 'view statistics page' do
+    visit statistics_path
```

Comment:
1º We need to create some sample data to be able to check something inside each accordion, for now, this spec is not checking calculations.
2º Given When Then (I missed some empty lines)